### DISCUSSION TITLE
Howto install on Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by Fact0ryy** [Ysurac/openmptcprouter#3106] [Q&A]
Hello dear team,
Hello Community,

I currently have a notebook with omptc running. Now I have an RP5 and Tee to move my setup. Unfortunately I can't find an ready for use image for it. Is there an image or how can I install it myself? If I have to compile it myself, which OS should I use? 

Looking forward to your answers.

BR

**Comment by Ysurac** [comment]
Images for Raspberry PI 5 are available here: https://snapshots.openmptcprouter.com/6.1/rpi5/targets/bcm27xx/bcm2712/
(It's still in snapshot, not yet a release)
